observability/pillars.md

```diff
@@ -47,8 +47,32 @@ Produces the information required to observe series of correlated operations in
 
 ## Metrics
 
-Metrics provide a near real-time indication of how the system is running. As opposed to logs and traces, the amount of data collected using metrics remains constant as the system load increases.
+Metrics provide a near real-time stream of data, informing operators and stakeholders about the functions the system is performing as well as its health. Unlike logging & tracing, metric data tends to be more efficient to transmit and store.
```

Comment:
about the functions the system is performing -> about how the functions in the system are performing

Reply:
Thanks for the feedback here. This part was purposefully about what the component/system is doing (performing an action), vs how well the action was performed. Both are part observability measures, and I'll find a way in a future update to incorporate perf measurement wording in the summary.